Demo/Demo/SwiftUIComponents/PayPalWebViews/PayPalTransactionButtonsView.swift

```diff
@@ -30,6 +30,15 @@ struct PayPalTransactionButtonsView: View {
                     .padding(5)
             )
             PayPalWebApprovalView(paypalWebViewModel: paypalWebViewModel)
+            if paypalWebViewModel.state.checkoutResult != nil {
+                NavigationLink {
+                    PayPalWebOrderCompletionView(orderID: orderID, payPalWebViewModel: paypalWebViewModel)
+                } label: {
+                    Text("Complete Order Transaction")
+                }
+                .buttonStyle(RoundedBlueButtonStyle())
+                    .padding()
```

Comment:
Should this indent be back a bit or should it be in the `.buttonStyle`? Not sure which element it applies to based on the spacing.

Reply:
it is a modifier for the NavigationLink which is pretty much a button. My indentation is off. I will fix it.

Reply:
Addressed here: 293097be913c95f2564c5f0fe0805570d66a9b4e